Source/ZoomNet/Resources/Phone.cs

```diff
@@ -62,6 +63,37 @@ public Task<PhoneCallUserProfile> GetPhoneCallUserProfileAsync(string userId, Ca
 				.AsObject<PhoneCallUserProfile>();
 		}
 
+		/// <inheritdoc cref="IPhone.ListPhoneCallUserProfilesAsync" select="remarks" />
+		public Task<PhoneCallUserProfilesPaginationObject> ListPhoneCallUserProfilesAsync(
```

Comment:
If it turns out that you can indeed use PaginatedResponseWithToken<T>, as I suspect, you'll be able to use the following:
```suggestion
		public Task<PaginatedResponseWithToken<PhoneCallUserProfile>> ListPhoneCallUserProfilesAsync(
```